src/core/hle/service/apt/apt.cpp

```diff
@@ -920,6 +921,16 @@ void Module::APTInterface::CheckNew3DS(Kernel::HLERequestContext& ctx) {
     LOG_WARNING(Service_APT, "(STUBBED) called");
 }
 
+void Module::APTInterface::CheckNew3DSThingy(Kernel::HLERequestContext& ctx) {
+    IPC::RequestParser rp(ctx, 0x103, 0, 0); // 0x01030000
+    IPC::RequestBuilder rb = rp.MakeBuilder(2, 0);
+
+    rb.Push(RESULT_SUCCESS);
+    rb.Push(static_cast<u8>(Settings::values.is_new_3ds) + 1);
```

Comment:
```suggestion
    rb.Push(static_cast<u8>(Settings::values.is_new_3ds + 1));
```

Otherwise the expression promotes to an `int`